src/cognito.js

```diff
@@ -97,3 +98,21 @@ export async function removeUserFromGroup(username, group) {
     throw err;
   }
 }
+
+/**
+ * Return the name of the cognito user pool. Note that this requires elevated
+ * privileges for the AWS user the server is running under (specifically
+ * `cognito-idp:DescribeUserPool action`), but if you do have these it can be
+ * very useful for debugging.
+ */
+export async function getCognitoUserPoolName() {
+  if (!COGNITO_USER_POOL_ID) return `No cognito pool in use`
+  try {
```

Comment:
I'll read #731 next week, but my aim for including this info is because I have a set of cognito (nextstrain.org) usernames (as do a few of us) and you need to remember which pool the server is accessing to know which subset of users are valid. Of the commits added here it's not the most important, and since our common server IAM users don't have the permissions to even see the user pool name I may end up dropping the commit.

Reply:
For sure, my point is that there are effectively two entrypoints to Cognito and the server could theoretically be using two different user pools at the same time if `config.json` isn't configured properly. If you're using this info to decide which user to use on the login page, `COGNITO_USER_POOL_ID` isn't the right variable to inspect - it should be the OIDC/OAuth2 variables.

Reply:
I've dropped this commit for now. I'd still like to include the functionality however I don't have time right now to invest in understanding the recent changes here. I'll revisit it some time in the future.

Reply:
Dropped commit for future reference: 3ef40529c311e9d7a3d27faccfdcf60b997ed8b3